feat(content): Add three more human starts (#11302)
endless-sky/endless-sky: data/human/intro missions.txt

	choice
		`	"I can't wait to explore the galaxy!"`
			goto galaxy
		`	"Just you wait; I'll be flying an entire fleet soon."`
			goto bigger
		`	"If I wanted to talk, I would have said something. Beat it."`
			decline
	label galaxy
	`	He grins. "Well, don't be too hasty. The galaxy is a big and dangerous place, and not for the fainthearted. It'll be quite a while until you can make the universe your oyster. Took me decades."`
		goto captain
	
	label syndicate
	`With a strange sense of relief, you give the salesman your money and jump into your new ship. The flight from the shipyard to the landing bays has you navigate through tunnels with only centimeters of clearance in all directions. After reaching the safety of the bay and landing, you disembark, where an old man in grease-stained coveralls is waiting.`
	`	"Nothing quite like buying your first starship," he says.`
	choice
		`	"I'm just glad to be free from working for someone else."`
			goto freedom
		`	"And there's nothing quite like going six hundred thousand credits into debt."`
			goto money
		`	"If I wanted to talk with you, I'd have said something. Leave me alone."`
			decline

> 	label freedom

Reviewer: ```suggestion
		`	"And there's nothing quite like going six hundred thousand credits into debt."`
			goto money
		`	"If I wanted to talk with you, I'd have said something. Leave me alone."`
			decline
```

		accept

conversation "freighter 1 end"
	`You drop off the shipment of <commodity> on <planet>, and collect your payment of <payment>. "Hauling freight," says James. "Not the most glamorous job, or the most interesting, but you're always guaranteed steady income. And once you know the galaxy a bit better, instead of taking missions you can find your own trade routes to follow. The only trouble is, build up a fleet with lots of cargo space and not enough defenses and the pirates begin to take notice."`
	choice
		`	"Did you have a lot of trouble with pirates when you were a captain?"`
		`	"But what if I want to earn money more quickly than I can by trading?"`
			goto quick
	`	"Some. That's why you can't just go traipsing around the galaxy in a fleet of unarmed star barges; every pirate within a week's travel will hear you're coming through and come after you. In the long term, having a few heavy freighters with good defenses is more cost-effective than losing a star barge or two to the pirates on every journey."`
		goto port
	label quick

> 	`	"There's no quick road to riches, kid. The more something pays, the riskier it is. If you need to make lots of money in a hurry, bounty hunting is the way to go, but nine out of ten pilots who give bounty hunting a try end up dead. And, no one's going to give you those sorts of missions until you've proved yourself in combat, anyway."`

Reviewer 2: ```suggestion
	`	"There's no quick road to riches, kid. The more something pays, the riskier it is. If you need to make lots of money in a hurry, bounty hunting is the way to go, but nine out of ten pilots who give bounty hunting a try end up dead. And no one's going to give you those sorts of missions until you've proved yourself in combat, anyway."`
```

	label port
	`	He picks up his suitcase. "If you want to run another mission with me, look for me in the spaceport. I'll see if I can find you something with a higher profit margin this time."`
	``
	`	(As before, click the "Spaceport" button to continue this mission.)`

conversation "freighter 2 end"
	`By now, your entire ship smells like brine and raw fish, but as best you can tell the fish is still fresh enough to eat. The dock workers on <planet> offload it without comment, and give you your payment of <payment>.`

> 		to display

Reviewer 2: Why do these lines differ depending on your start? To be honest I don't really see why they would be different.

	`	After <they are> gone, James turns to you and says, "Not too bad for your first escort mission. I recognize that look on your face - it's the same one I had after my first job like this back when I was a beginner. If I recall, my first priority after touching down was replacing some of the ship's guns with missile launchers so I'd actually have a chance against pirates. It's a scandal how they sell interceptors with nothing but short-ranged energy weapons these days; trying to use them in combat on such a small ship is practically suicide. Keep your distance with some missiles and you can almost never go wrong.`

	`	"Anyway, I'll stop rambling. If you've still got a hankering for some dangerous missions, come and find me in the spaceport before you leave." He picks up his suitcase and heads outside.`
		to display
			has "start: default"
	`	"Anyway, I'll stop rambling." James pauses for a moment, then says, "I've got an idea for something you might want to try. Meet me in the spaceport bar in a little while."`
		to display
			has "start: deep"
	``
		to display
			or
				has "start: default"
				has "start: paradise"

> 				has "start: syndicate"

Author: It's a conditional line break that will only show up if one of the lines beneath it appears.

	source "New Boston"
	destination "New Greenland"
	to offer
		has "start: default"
	substitutions
		<trade> "stock up on medical goods"

> 		<while> "While they don't sell much here compared to on the big manufacturing worlds"

Reviewer 3: ```suggestion
		"<trade>" "stock up on medical goods"
		"<while>" "While they don't sell much here compared to on the big manufacturing worlds"
```

			``
			`	(As before, click the "Spaceport" button to continue this mission. The next mission will involve carrying cargo, so it will only show up if you have all 15 tons of space free.)`

mission "Delve Intro [2 Interceptor]"
	priority
	name "Delivery to <planet>"
	description "Deliver <commodity> to <destination>, while avoiding pirate attacks."
	blocked `When you meet up with James in the spaceport, he says, "Hi there, <first>! I found you another mission, but it looks like you'll need <capacity> in order to take it on. So, you'll either have to cancel another mission, or come back here later to continue this one. Sorry!"`
	passengers 1
	cargo niobium 15
	source "Lodestone"
	destination "Ada"
	to offer
		has "Delve Intro [1 Interceptor]: done"
	npc
		government Pirate
		personality entering plunders
		ship "Splinter" "Spear of Heimdall"
	
	on offer
		conversation
			`You find James in the spaceport bar, talking with a man wearing a business suit. The expression on James' face is growing increasingly concerned as he speaks. Seeing you come in, he waves to you and you join them at the table.`

> 			`	"Hello <first>," says James. "This is Miguel, an independent mine owner of <origin>." Miguel smiles at you.`

Reviewer 2: ```suggestion
			`	"Hello <first>," says James. "This is Miguel, an independent mine owner on <origin>." Miguel smiles at you.`
```

			`	You find James waiting for you inside the spaceport bar. Next to him is a man with a half-eaten hamburger and a plate of fries, busily examining a star map of the local sector. He notices your arrival and stands up to greet you. "<first>, this is Captain August," James says as you shake his hand. "I told him about your new ship and how it'd be a good idea for you to get a taste of asteroid mining. He mines in this part of space and says there's a good system nearby if you're willing to give it a try."`
			choice
				`	"I'll give it a shot."`
					goto mining
				`	"I think I'm ready to head out on my own."`
					decline
			

> 			label "mining"

Reviewer 2: ```suggestion
			label mining
```

		government Pirate
		personality entering plunders
		ship "Bastion" "Hammer of Thor"
	substitutions
		<person> "she"
		<enemy> "Hammer"

> 		<magnitude> "ten"

Reviewer 3: ```suggestion
		"<person>" "she"
		"<enemy>" "Hammer"
		"<magnitude>" "ten"
```

	
	on offer
		conversation "interceptor evade intro"
	
	on complete
		payment 45000
		conversation "interceptor evade end"

mission "Intro [2.1 Interceptor]"
	priority
	name "Mine asteroids in <system>"
	description "Go to the <system> system and mine five tons of silicon, then land on <planet>."
	blocked `When you meet up with James in the spaceport, he says, "Hi there, <first>! I think you'll find what I want to show you next very interesting, but you'll need to free up <capacity> first. So, you'll either have to cancel another mission, or come back here later to continue this one. Sorry!"`
	passengers 1
	source "Ingot"
	destination "New Sahara"
	to offer
		has "Intro [2 Interceptor]: done"
		not "Intro [3 Interceptor]: offered"
	substitutions
		<captain> "Marshall"

> 		<they> "he"

Reviewer 3: ```suggestion
		"<captain>" "Marshall"
		"<they>" "he"
```

			label end
			``
			`	(As before, click the "Spaceport" button to continue this mission.)`

mission "Midgard Intro [2 Interceptor]"
	priority
	name "Escort convoy to <planet>"
	description "Escort a merchant convoy to <destination>, along with James."
	blocked `When you meet up with James in the spaceport, he says, "Hi there, <first>! I found you another mission, but it looks like you'll need <capacity> in order to take it on. So, you'll either have to cancel another mission, or come back here later to continue this one. Sorry!"`
	passengers 1
	source "Valhalla"
	destination "Ingot"
	to offer
		has "Midgard Intro [1 Interceptor]: done"
	npc
		government Pirate
		system "Tania Australis"
		personality plunders
		ship "Sparrow" "Beast of Brass"
		ship "Berserker" "Bootlegger"
	npc save
		government Merchant
		personality timid escort
		ship "Freighter" "Conagher's Porter"
		ship "Freighter" "Solemn Vow"
	substitutions
		<their> "their"
		<they're> "they're"

> 		"<they are>" "they are"

Reviewer 3: ```suggestion
		"<their>" "their"
		"<they're>" "they're"
		"<they are>" "they are"
```

			`	"Also, make sure to refuel your ship before jumping into Algieba; if you run out of fuel there, you'll be a sitting duck unless you can get another ship to help you."`
				accept
	
	on visit
		dialog phrase "generic arrived-without-npc dialog"
	on complete
		payment 50000
		conversation "interceptor escort end"

mission "Midgard Intro [2.1 Interceptor]"
	priority
	name "Mine asteroids in <system>"
	description "Go to the <system> system and mine five tons of silicon, then land on <planet>."
	blocked `When you meet up with James in the spaceport, he says, "Hi there, <first>! I think you'll find what I want to show you next very interesting, but you'll need to free up <capacity> first. So, you'll either have to cancel another mission, or come back here later to continue this one. Sorry!"`
	passengers 1
	source "Ingot"
	destination "New Sahara"
	to offer
		has "Midgard Intro [2 Interceptor]: done"
	substitutions
		<captain> "Marshall"

> 		<they> "he"

Reviewer 3: ```suggestion
		"<captain>" "Marshall"
		"<they>" "he"
```

	blocked `When you meet up with James in the spaceport, he says, "Hi there, <first>! I have one final mission for you, but you'll need <capacity> in order to take it on. So, you'll either have to cancel another mission, or come back here later to continue this one. Sorry!"`
	passengers 1
	source "New Sahara"
	destination "Hestia"
	to offer
		has "Midgard Intro [2.1 Interceptor]: done"
	
	on offer
		conversation "mining intro new sahara"
	
	on complete
		payment 50000
		log "Dropped James off on Hestia after running a few missions with him. Escort duty is a risky business."
		log "Minor People" "James" `He currently resides on Hestia, having retired from life as a merchant captain.`
		conversation "end of intro missions"

# Missions for the Mainsail start

mission "Mainsail Intro [0]"
	name "Passenger to <planet>"
	description "This old-timer captain offered to ride along with you to <destination>, and to give you some tips along the way."
	landing
	passengers 1
	source "Mainsail"
	destination "Glory"
	to offer
		has "start: paradise"
	substitutions
		<trade> "load some electronic goods onto your ship"

> 		<while> "While they don't sell much here compared to on the big manufacturing worlds"

Reviewer 3: ```suggestion
		"<trade>" "load some electronic goods onto your ship"
		"<while>" "While they don't sell much here compared to on the big manufacturing worlds"
```

		log "Minor People" "James" `He currently resides on Hestia, having retired from life as a merchant captain.`
		conversation "end of intro missions"

mission "Mainsail Intro [0.1 Interceptor]"
	priority
	name "Mine asteroids in <system>"
	description "Go to the <system> system and mine five tons of lead, then land on <planet>."
	blocked `When you meet up with James in the spaceport, he says, "Hi there, <first>! I think you'll find what I want to show you next very interesting, but you'll need to free up <capacity> first. So, you'll either have to cancel another mission, or come back here later to continue this one. Sorry!"`
	passengers 1
	source "Glory"
	destination "Pearl"
	to offer
		has "Mainsail Intro [0]: done"
		or
			has "ships: Interceptor"
			has "ships: Drone"
			has "ships: Fighter"
			has "ships: Light Warship"
			has "ships: Medium Warship"
			has "ships: Heavy Warship"
			has "ships: Superheavy"
		not "Mainsail Intro [1 Transport]: offered"
		not "Mainsail Intro [1 Transport Alt]: offered"
		not "Mainsail Intro [1 Freighter]: offered"
	substitutions
		<captain> "Hwang"

> 		<they> "she"

Reviewer 3: ```suggestion
		"<captain>" "Hwang"
		"<they>" "she"
```

			label end
			choice
				`	"Of course."`
				`	"Sorry, I think I'll go hunt some pirates instead."`
					decline
			`	Once again you walk back to your ship with James. You've heard of <planet> before; one of the Paradise Worlds terraformed from a lifeless ice planet to the largest and most luxurious retirement home in the galaxy. As you look at his utilitarian outfit and modest luggage, something seems off about the idea of James living in such a place.`
				accept
	
	on complete
		payment 30000
		log "Dropped James off on Hestia after running a few missions with him. Escort duty is a risky business."
		log "Minor People" "James" `He currently resides on Hestia, having retired from life as a merchant captain.`
		conversation "end of intro missions"

# Missions for the Delve start

mission "Delve Intro [0]"
	name "Passenger to <planet>"
	description "This old-timer captain offered to ride along with you to <destination>, and to give you some tips along the way."
	landing
	passengers 1
	source "Delve"
	destination "Burthen"
	to offer
		has "start: syndicate"
	substitutions
		<trade> "stock up on medical goods"

> 		<while> "While most of the outfits will be far outside your price range"

Reviewer 3: ```suggestion
		"<trade>" "stock up on medical goods"
		"<while>" "While most of the outfits will be far outside your price range"
```

			`	After taking one last glance out the window, he says, "Anyway, we're nearing my destination. Are you willing to take me on one last trip? I need to get to <destination>, and of course I'll pay you quite well to take me there."`
			choice
				`	"Of course."`
				`	"Sorry, but I was planning on heading out in a different direction from here."`
					decline
			`	Once again you walk back to your ship with James. Once you get there, as he's stowing his luggage, you pull up the information your computer has on <planet>. It's one of the "Paradise Worlds," terraformed to have a perfect climate, accessible only to the very wealthy. You wonder whether James is going to be able to fit in there.`
				accept
	
	on complete
		payment 10000

> 		log "Dropped James off on Hestia after running a few missions with him. Met a few spoiled tourists and revisited Earth along the way."

Author: ```suggestion
		log "Dropped James off on Hestia after running a few missions with him. Met a few tourists and revisited Earth along the way."
```

		conversation "end of intro missions"

mission "Delve Intro [1 Interceptor]"
	priority
	name "Escort convoy to <planet>"
	description "Escort a merchant convoy to <destination>, along with your new friend James."
	blocked `When you meet up with James in the spaceport, he says, "Hi there, <first>! I found you another mission, but it looks like you'll need <capacity> in order to take it on. So, you'll either have to cancel another mission, or come back here later to continue this one. Sorry!"`
	passengers 1
	source "Burthen"
	destination "Lodestone"
	to offer
		has "Delve Intro [0]: done"
		or
			has "ships: Interceptor"
			has "ships: Drone"
			has "ships: Fighter"
			has "ships: Light Warship"
			has "ships: Medium Warship"
			has "ships: Heavy Warship"
			has "ships: Superheavy"
		not "Delve Intro [1 Transport]: offered"
		not "Delve Intro [1 Freighter]: offered"
	npc
		government Pirate
		personality entering plunders
		ship "Sparrow" "Vain Butcher"
		ship "Fury" "Rotten Clout"
	npc save
		government Merchant
		personality timid escort
		ship "Freighter" "Spring Coming"
		ship "Freighter" "Paparazzi Party"
	substitutions
		<their> "her"
		<they're> "she's"

> 		"<they are>" "she is"

Reviewer 3: ```suggestion
		"<their>" "her"
		"<they're>" "she's"
		"<they are>" "she is"
```

	passengers 1
	cargo "niobium" 15
	source "Lodestone"
	destination "Ada"
	to offer
		has "Delve Intro [1 Interceptor]: done"
	npc
		government Pirate
		personality entering plunders
		ship "Splinter" "Spear of Heimdall"
	substitutions
		<person> "he"
		<enemy> "Spear"
		<magnitude> "ten"
	
	on offer
		conversation "interceptor evade intro"
	
	on complete
		payment 35000
		conversation "interceptor evade end"

mission "Delve Intro [2.1 Interceptor]"
	priority
	name "Mine asteroids in <system>"
	description "Go to the Procyon system and mine five tons of silicon, then land on <planet>."
	blocked `When you meet up with James in the spaceport, he says, "Hi there, <first>! I think you'll find what I want to show you next very interesting, but you'll need to free up <capacity> first. So, you'll either have to cancel another mission, or come back here later to continue this one. Sorry!"`
	passengers 1
	source "Ada"
	destination "Relic"
	to offer
		has "Delve Intro [2 Interceptor]: done"
	substitutions
		<captain> "August"

> 		<they> "he"

Reviewer 3: ```suggestion
		"<captain>" "August"
		"<they>" "he"
```

		has "Intro [2.1 Interceptor]: done"
	
	on offer
		conversation "mining intro new sahara"
	
	on complete
		payment 30000
		log "Dropped James off on Hestia after running a few missions with him. Escort duty is a risky business."
		log "Minor People" "James" `He currently resides on Hestia, having retired from life as a merchant captain.`
		conversation "end of intro missions"

# Missions for the Midgard start

mission "Midgard Intro [0]"
	name "Passenger to <planet>"
	description "This old-timer captain offered to ride along with you to <destination>, and to give you some tips along the way."
	landing
	passengers 1
	source "Midgard"
	destination "Asgard"
	to offer
		has "start: deep"
	substitutions
		<trade> "load some bundles of clothing onto your ship"
		<while> "While most of the outfits will be far outside your price range"
	
	on offer
		conversation "intro mission"
	
	on accept
		log "Getting some basic instructions on making a living as a merchant from an old captain named James."

> 		log "Minor People" "James" `A old man who accompanied you on your first few jobs as a starship captain.`

Reviewer: ```suggestion
		log "Minor People" "James" `An old man who accompanied you on your first few jobs as a starship captain.`
```

	passengers 2
	source "Norn"
	destination "Earth"
	to offer
		has "Midgard Intro [1 Transport Alt]: done"
	
	on offer
		conversation
			`The smell of brine permeates <origin>'s spaceport. By the time you find James, it seems unlikely that you'll ever eat seafood without gagging again. "Hi there, Captain <last>!" he says. "Interested in carrying a returning tourist to <planet>?" Behind him is a young man in red sunglasses.`
			choice
				`	"Sure!"`
				`	"No thanks, I think I'll strike off on my own now."`
					decline
			`	"Great," he says. "This is Van. Let's help him carry his luggage to your ship." As you begin walking back, slightly ahead of them, James whispers to you, "Tourists are always a good way to make money, especially if you're low on bunk space. They're on vacation, so they don't mind paying a bit extra. But no point in your cargo space going to waste, so take a look at the job board and see if there are any delivery missions we can run at the same time."`
				accept
	
	on complete
		payment 22000
		conversation

> 			`As you make the final landing descent to the surface of Earth, Van looks down on Earth's cluttered surface and begins listing off places he recognizes. James joins in by bringing up the history of the cities you fly over, and the two of them begin making joking about the worst places to live in on <planet>.`

Reviewer: ```suggestion
			`As you make the final landing descent to the surface of Earth, Van looks down on Earth's cluttered surface and begins listing off places he recognizes. James joins in by bringing up the history of the cities you fly over, and the two of them begin joking about the worst places to live on <planet>.`
```
alternate
```suggestion
			`As you make the final landing descent to the surface of Earth, Van looks down on Earth's cluttered surface and begins listing off places he recognizes. James joins in by bringing up the history of the cities you fly over, and the two of them begin making jokes about the worst places to live on <planet>.`
```

			label end
			``
			`	(As before, click the "Spaceport" button to continue this mission.)`

mission "Midgard Intro [2 Interceptor]"
	priority
	name "Escort convoy to <planet>"
	description "Escort a merchant convoy to <destination>, along with James."
	blocked `When you meet up with James in the spaceport, he says, "Hi there, <first>! I found you another mission, but it looks like you'll need <capacity> in order to take it on. So, you'll either have to cancel another mission, or come back here later to continue this one. Sorry!"`
	passengers 1
	source "Valhalla"
	destination "Ingot"
	to offer
		has "Midgard Intro [1 Interceptor]: done"
	npc
		government Pirate
		system "Tania Australis"
		personality plunders
		ship "Sparrow" "Beast of Brass"
		ship "Berserker" "Bootlegger"
	npc save
		government Merchant
		personality timid escort
		ship "Freighter" "Conagher's Porter"
		ship "Freighter" "Solemn Vow"
	substitutions
		<their> "their"
		<they're> "they're"
		"<they are>" "they are"
	
	on offer
		conversation

> 			`The landscape of <origin> is beyond even the urban sprawls of Midgard and Asgard. Beyond the spaceport's glass walls is a vast forest of skyscrapers reaching out to the edge of your vision. Far away, the lights of distant buildings blend together, as though the horizon is ablaze.`

Reviewer: ditto about beyond

			`	You find James standing on a landing pad underneath a large freighter, chatting with a person you assume is the captain. "There you are!" he says. "Captain McLain here says their two ships have always had troubles with pirates when venturing outside of the Deep, and they're looking for an escort to get them safely to Gemstone. Are you interested?"`
			choice
				`	"Of course!"`
				`	"No, sounds a bit too risky for me."`
					decline

> 			`	"Great," he says. "Whenever we're ready to leave the planet, they'll take off behind us and follow us to <planet>." As you begin walking away together, he adds in a quieter voice, "While the Deep is well guarded against pirates, most of human space is clamoring with bandits and other ruffians. Normally a fresh pilot like you with no combat reputation would never be offered a mission like this, so try not to mess things up. Remember that we don't have to kill the pirates, all we have to do is make sure the convoy isn't destroyed. But freighters are slow, so you can't just go skipping through hyperspace as fast as you want; you need to stick around and make sure they're behind you.`

Author: `"Intro [1 Interceptor]"` (the existing escort intro mission) uses "whenever" here.

			`	"Also, make sure to refuel your ship before jumping into Algieba; if you run out of fuel there, you'll be a sitting duck unless you can get another ship to help you."`
				accept
	
	on visit
		dialog phrase "generic arrived-without-npc dialog"
	on complete
		payment 50000
		conversation "interceptor escort end"

mission "Midgard Intro [2.1 Interceptor]"
	priority
	name "Mine asteroids in <system>"

> 	description "Go to the <system> system and mine 5 tons of Silicon, then land on <planet>."

Author: This is how the existing tutorial mining mission is already formatted.

	blocked `When you meet up with James in the spaceport, he says, "Hi there, <first>! I think you'll find what I want to show you next very interesting, but you'll need to free up <capacity> first. So, you'll either have to cancel another mission, or come back here later to continue this one. Sorry!"`
	passengers 1
	source "Ingot"
	destination "New Sahara"
	to offer
		has "Midgard Intro [2 Interceptor]: done"
	
	on offer
		conversation "mining intro ingot"
	
	on visit
		dialog `You've reached <destination>, but you don't have at least five tons of silicon in your cargo hold. Locate and mine silicon asteroids in this system before returning.`
	
	on complete
		require "Silicon" 5

> 		conversation "mining intro end"

Reviewer: Shouldn't the silicon be taken, or does the player need to sell it manually? Might be worth having James mention it if the latter is the case.

			`	You say goodbye to them and wish them well, and as agreed they pay you <payment>. You have just completed your first proper job as a starship captain!`
			`	As James leaves the ship, he says, "Again, if you're okay with continuing to give me a ride, just meet me in the spaceport."`
			``
			`	(As before, click the "Spaceport" button to continue this mission.)`

mission "Mainsail Intro [2 Transport]"
	priority
	name "Tourists to <planet>"
	description "These tourists will pay you <payment> to take them to <destination>. Your new friend James will also be tagging along for the ride."
	blocked `When you meet up with James in the spaceport, he says, "Hi there, <first>! I found you another mission, but it looks like you'll need <capacity> in order to take it on. So, you'll either have to cancel another mission, or come back here later to continue this one. Sorry!"`
	passengers 4
	source "Tinker"
	destination "Earth"
	to offer
		has "Mainsail Intro [1 Transport]: done"
	
	on offer
		conversation
			`It is raining.`

> 			`	A deep fog envelops the spaceport city on <origin>. In the distance, you can see the faded lights of distant starships drifting like phantoms. On the streets below, harsh rain whips against the congested traffic, blurring it into a muddy mess of red light.`

Author: For effect.

			choice
				`	"Somebody has to do the dangerous stuff so that the rest of humanity can prosper."`
				`	"It does seem awfully isolated. But some people might enjoy the solitude."`
			`	"I suppose," he says. "Anyway, I think it's time for the last leg of my journey. Do you think you can give me a lift to <planet>? I'll pay you quite well for your services, of course."`
			choice
				`	"I can take you there."`
				`	"Sorry, I was thinking of heading in a different direction from here."`
					decline
			`	Once again you walk back to your ship with James. You've heard of <planet> before; one of the Paradise Worlds terraformed from a lifeless ice planet to the largest and most luxurious retirement home in the galaxy. As you look at his utilitarian outfit and modest luggage, something seems off about the idea of James living in such a place.`
				accept
	
	on complete
		payment 30000
		log "Dropped James off on Hestia after running a few missions with him. Still not convinced that my hold isn't a biohazard."
		log "Minor People" "James" `He currently resides on Hestia, having retired from life as a merchant captain.`
		conversation "end of intro missions"

> mission "Mainsail Intro [0.1 Interceptor]"

Reviewer: It occurs to me that for these mining missions you could have another conversation. Use substitutions for the names of the non-James characters and the first paragraph, and you'll cut down on quite a bit of duplication.

			`	"Captain <last>!" he says. "Wonderful to see you again. I was just talking to this family. They've been having trouble finding a single ship that can give them all passage, but I told them that your shuttlecraft has plenty of bunks. They can pay you <payment> to take them to <planet>, which is just a few jumps away from Earth, and is a planet well worth visiting in its own right. What do you say?"`
			`	The husband chimes in. "We were afraid that if we were split across multiple ships, we wouldn't be able to keep an eye on Faiza." He gestures towards the child in the pram. "She hasn't been taking Burthen's gravity well."`
			choice
				`	"Welcome aboard!"`
				`	"Sorry, I think I'd rather go off in my own direction from here."`
					decline
			`	You and James direct the team to your ship. Looks like you're about to set off on your first real job...`
				accept
	
	on complete
		payment 30000
		conversation

> 			`Along the journey, you learned more about the family you are transporting and why they are moving to <planet>. "There's Faiza's condition, of course, but we've always wanted to move offworld since before our first child was born," the mother, Carlijn says. "My husband and I constantly moved between different mining towns as we grew up. We didn't want our children to go through the same thing."`

Reviewer: ```suggestion
			`Along the journey, you learned more about the family you are transporting and why they are moving to <planet>. "There's Faiza's condition, of course, but we've always wanted to move offworld since before our first child was born," says the mother, Carlijn. "My husband and I constantly moved between different mining towns as we grew up. We didn't want our children to go through the same thing."`
```

			`	After they leave, James chuckles. "Can't say I've ever seen an Earth fanatic and an Earth native in the same group before, let alone be on good terms. That's one of the perks of being a starship captain, though: you get to have the most unique experiences in the galaxy." Then, he picks up his luggage and adds, "Same deal as before. If you want to keep traveling together, meet me in the spaceport."`

mission "Delve Intro [3 Transport]"
	priority
	name "Bring James to <planet>"
	description "Bring James to <destination>, where he has a spot reserved in a retirement home."
	blocked `When you meet up with James in the spaceport, he says, "Hi there, <first>! I found you another mission, but it looks like you'll need <capacity> in order to take it on. So, you'll either have to cancel another mission, or come back here later to continue this one. Sorry!"`
	passengers 1
	source "Earth"
	destination "Hestia"
	to offer
		has "Delve Intro [2 Transport]: done"
	
	on offer
		conversation

> 			`<origin> is the same as it's always been. Homeless men wrapped in old army blankets beg money from well-dressed businessmen. Soldiers in Navy uniforms walk by in groups of ten or twenty, driving those beggars to hide behind trashcans or do their best impression of a respectable citizen until they pass. Starship captains haggle with merchants, speaking of sums greater than the income of some neighborhoods. Flashes of light come from various landing bays as poorly-protected welders risk their health to repair ships destined for places they can only dream of.`

Reviewer: ```suggestion
			`<origin> is the same as it's always been. Homeless men wrapped in old army blankets beg money from well-dressed businessmen. Soldiers in Navy uniforms walk by in groups of ten or twenty, driving the beggars to hide behind trashcans or do their best impressions of respectable citizens until they pass. Starship captains haggle with merchants, speaking of sums greater than the income of some neighborhoods. Flashes of light come from various landing bays as poorly-protected welders risk their health to repair ships destined for places they can only dream of.`
```

			choice
				`	"If they're paying, then I'm in."`
				`	"Sorry, but I think I'm going to head out on my own."`
					decline
			`	"Great!" says James. "They'll be loading up your ship shortly."`
				accept
	
	on complete
		payment 22000
		conversation "freighter 1 end"

mission "Delve Intro [2 Freighter]"
	priority
	name "Rush delivery to <planet>"
	description "This shipment of <commodity> needs to get to <destination>, by <date>."
	blocked `When you meet up with James in the spaceport, he says, "Hi there, <first>! I found you another mission, but it looks like you'll need <capacity> in order to take it on. So, you'll either have to cancel another mission, or come back here later to continue this one. Sorry!"`

> 	cargo thorium 33

Reviewer: ```suggestion
	cargo "thorium" 33
```

mission "Delve Intro [1 Interceptor]"
	priority
	name "Escort convoy to <planet>"
	description "Escort a merchant convoy to <destination>, along with your new friend James."
	blocked `When you meet up with James in the spaceport, he says, "Hi there, <first>! I found you another mission, but it looks like you'll need <capacity> in order to take it on. So, you'll either have to cancel another mission, or come back here later to continue this one. Sorry!"`
	passengers 1
	source "Burthen"
	destination "Lodestone"
	to offer
		has "Delve Intro [0]: done"
		or
			has "ships: Interceptor"
			has "ships: Drone"
			has "ships: Fighter"
			has "ships: Light Warship"
			has "ships: Medium Warship"
			has "ships: Heavy Warship"
			has "ships: Superheavy"
		not "Delve Intro [1 Transport]: offered"
		not "Delve Intro [1 Freighter]: offered"
	npc
		government Pirate
		personality entering plunders
		ship "Sparrow" "Vain Butcher"
		ship "Fury" "Rotten Clout"
	npc save
		government Merchant
		personality timid escort
		ship "Freighter" "Spring Coming"
		ship "Freighter" "Paparazzi Party"
	substitutions
		<their> "her"
		<they're> "she's"
		"<they are>" "she is"
	
	on offer
		conversation

> 			`Every step on <origin> is a struggle. By the time you reach the spaceport cafe, you have just enough energy to slump into a chair with steel legs as thick as its cushions. After taking a rest, you find James standing on a landing pad underneath a large freighter, chatting with a woman you assume is the captain. "There you are!" he says. "Captain Rawda here says her two ships have always had trouble with pirates in the Core, and she's looking for an escort to get them safely to <planet>. Are you interested?"`

Reviewer: > chatting with a woman you assume is the captain

How do you know there will be a captain? James doesn't mention it anywhere.

> 			`Every step on <origin> is a struggle. By the time you reach the spaceport cafe, you have just enough energy to slump into a chair with steel legs as thick as its cushions. After taking a rest, you find James standing on a landing pad underneath a large freighter, chatting with a woman you assume is the captain. "There you are!" he says. "Captain Rawda here says her two ships have always had trouble with pirates in the Core, and she's looking for an escort to get them safely to <planet>. Are you interested?"`

Author: Who else would he be talking to under a freighter?

	blocked `When you meet up with James in the spaceport, he says, "Hi there, <first>! I found you another mission, but it looks like you'll need <capacity> in order to take it on. So, you'll either have to cancel another mission, or come back here later to continue this one. Sorry!"`
	passengers 1
	cargo niobium 15
	source "Lodestone"
	destination "Ada"
	to offer
		has "Delve Intro [1 Interceptor]: done"
	npc
		government Pirate
		personality entering plunders
		ship "Splinter" "Spear of Heimdall"
	substitutions
		<person> "he"
		<enemy> "Spear"
		<magnitude> "ten"
	
	on offer
		conversation "interceptor evade intro"
	
	on complete
		payment 35000
		conversation "interceptor evade end"

mission "Delve Intro [2.1 Interceptor]"
	priority
	name "Mine asteroids in <system>"
	description "Go to the Procyon system and mine 5 tons of Silicon, then land on <planet>."
	blocked `When you meet up with James in the spaceport, he says, "Hi there, <first>! I think you'll find what I want to show you next very interesting, but you'll need to free up <capacity> first. So, you'll either have to cancel another mission, or come back here later to continue this one. Sorry!"`

> 	passengers 1

Reviewer: ```suggestion
	passengers 1
	mark "Procyon"
```

Author: Any reason for moving the mark and not the `on enter` that removes the mark?

	blocked `When you meet up with James in the spaceport, he says, "Hi there, <first>! I found you another mission, but it looks like you'll need <capacity> in order to take it on. So, you'll either have to cancel another mission, or come back here later to continue this one. Sorry!"`
	passengers 1
	source "Relic"
	destination "Hestia"
	to offer
		has "Delve Intro [2.1 Interceptor]: done"
	
	on offer
		conversation "intro relic"
	
	on complete
		payment 10000
		log "Dropped James off on Hestia after running a few missions with him. Escort duty is a risky business."
		log "Minor People" "James" `He currently resides on Hestia, having retired from life as a merchant captain.`
		conversation "end of intro missions"

substitutions
	"<intro end suit>" ""
	"<intro end suit>" " As a lifelong textile worker, you automatically note that the suit is made of real wool, very finely woven, and that it has been tailored to fit him."
		has "start: default"
	"<intro end suit>" " You recognize a logo on the collar of the suit that belongs to a famous brand."
		has "start: paradise"

conversation "end of intro missions"

> 	`As you are piloting your ship down to land on <planet>, James disappears into his bunk room. After the ship has landed he comes out, no longer in the grease-stained coveralls he's been wearing up until now, but in a suit.<intro end suit> He smooths down the front of the suit distractedly and then looks at his hands as if unsure of where to put them. "How do I look?" he asks.`

Reviewer 2: He disappears into his bunk frame?
